toolkit/tools/scheduler/scheduler.go

```diff
@@ -210,6 +210,19 @@ func main() {
 			logger.Log.Warnf("Failed to initialize the ccache manager:\n%v", err)
 		}
 	}
+
+	if *noCleanup {
+		for _, line := range logger.FormatWarningBox([]string{
+			"'--no-cleanup' requested. Build agent directories will not be removed automatically.",
+			"(" + *workDir + "/*)",
+			"Manual cleanup is required.",
+			"Use 'make clean-build-packages-workers' to remove build agent directories.",
+			"",
+			"Also consider using 'make containerized-rpmbuild' to debug package build issues",
+		}) {
+			logger.Log.Warn(line)
```

Comment:
Expected behaviour, I'd suggest using info for this one. Maybe an additional `ATTENTION!` line to bring more attention to it, if needed.

Reply:
```
INFO[0040][scheduler] ╔══════════════════════════════════════════════════════════════════════════════════════╗ 
INFO[0040][scheduler] ║                                      ATTENTION!                                      ║ 
INFO[0040][scheduler] ║                                                                                      ║ 
INFO[0040][scheduler] ║ '--no-cleanup' requested. Build agent directories will not be removed automatically. ║ 
INFO[0040][scheduler] ║         (/home/damcilva/repos/temp/CBL-Mariner_TEMP5/build/worker/chroot/*)          ║ 
INFO[0040][scheduler] ║                             Manual cleanup is required.                              ║ 
INFO[0040][scheduler] ║      Use 'make clean-build-packages-workers' to remove build agent directories.      ║ 
INFO[0040][scheduler] ║                                                                                      ║ 
INFO[0040][scheduler] ║   Also consider using 'make containerized-rpmbuild' to debug package build issues    ║ 
INFO[0040][scheduler] ╚══════════════════════════════════════════════════════════════════════════════════════╝ 
```